src/commands/logs.rs

```diff
@@ -0,0 +1,168 @@
+use std::ops::Sub;
+use std::time::Duration;
+
+use anyhow::{anyhow, Result};
+use chrono::Utc;
+use cloud_openapi::models::Entry;
+
+use cloud::{client::Client as CloudClient, CloudClientExt, CloudClientInterface};
+
+use crate::commands::create_cloud_client;
+use crate::commands::deploy::SPIN_DEPLOY_CHANNEL_NAME;
+use crate::opts::*;
+use clap::Parser;
+use uuid::Uuid;
+
+/// Fetch and tail logs of an application deployed to the Fermyon Cloud.
+#[derive(Parser, Debug)]
+#[clap(about = "fetch logs for an app from Fermyon Cloud")]
+pub struct LogsCommand {
+    /// Find app to fetch logs for in the Fermyon instance saved under the specified name.
+    /// If omitted, Spin looks for app in default unnamed instance.
+    #[clap(
+        name = "environment-name",
+        long = "environment-name",
+        env = DEPLOYMENT_ENV_NAME_ENV
+    )]
+    pub deployment_env_id: Option<String>,
+
+    /// App name
+    #[clap(name = "app-name", long = "app-name")]
+    pub app_name: String,
```

Comment:
To be consistent with other cases we probably do not want to have an argument (eg) 'apps info' and `apps delete` commands. This would change to 

```bash
spin cloud logs <APP_NAME>
```

```suggestion
    pub app_name: String,
```

Reply:
If this is the same as the app name in `spin cloud link` and `spin cloud variables` then it should be `#[clap(short = 'a', long = "app")]` (actually we are not 100% consistent about having a `short` form but the long form is always "app").

Reply:
thanks. renamed variable to `app` and changed it to be an argument instead of flag